src/Configuration/AccountManager.cs

```diff
@@ -9,42 +10,64 @@ internal static class AccountManager
     {
         public static List<Account> Accounts;
 
-        private static void Load(string serverName)
+        public static string[] GetAccountNames(string serverName)
         {
-            if (Accounts == null)
+            try
             {
-                var accounts = LoadAccountsFromFile();
-                Accounts = accounts.Where(x => x.Server == serverName).ToList();
+                Load(serverName);
+                return Accounts?.Where(x => x.Server == serverName).Select(x => x.UserName).ToArray() ?? new string[] { };
+            }
+            catch(Exception ex)
+            {
+                //Log error?
+                return new string[] { };
             }
-        }
-
-        public static string[] GetAccountNames(string serverName)
-        {
-            Load(serverName);
-            return Accounts?.Where(x => x.Server == serverName).Select(x => x.UserName).ToArray() ?? new string[] { };
         }
 
         public static string GetAccountPassword(string serverName, string userName)
         {
-            Load(serverName);
-            return Accounts?.FirstOrDefault(x => x.UserName == userName)?.Password;
+            try
+            {
+                Load(serverName);
+                return Accounts?.FirstOrDefault(x => x.UserName == userName)?.Password;
+            }
+            catch(Exception ex)
+            {
+                //Log Error?
```

Comment:
Accounts dont have to have a password, if i remember correctly, so no error, technically, right?

Reply:
correct, this was again in regards to the file operation. If the try/catch is moved closer this would make sense.